nightingale_proprioception/CMakeLists.txt

```diff
@@ -0,0 +1,207 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(nightingale_proprioception)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  geometery_msgs
```

Comment:
I think this should be geometry. Did this build successful with this? Or is this not a typo?

Reply:
also in other lines if this is indeed a typo